WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift

```diff
@@ -269,3 +269,21 @@ private extension SignupEpilogueViewController {
         static let primary = NSLocalizedString("Continue", comment: "Button text on site creation epilogue page to proceed to My Sites.")
     }
 }
+
+// MARK: - User Defaults
+
+@objc
+extension UserDefaults {
+    private enum Keys: String {
+        case quickStartWasDismissedPermanently = "QuickStartWasDismissedPermanently"
+    }
+
+    var quickStartWasDismissedPermanently: Bool {
```

Comment:
Hey @nheagy , you could also use #function instead of defining the key:
var quickStartWasDismissedPermanently: Bool {
        get { return bool(forKey: #function) }
        set { set(newValue, forKey: #function) }
    }
Using #function defines the saved key as the name of the variable

Reply:
TIL! We've used this exact pattern multiple places, and have never use `#function`. Seems cool, I'm going to try it 👍